### PR TITLE
Deploy StakingProxy, LockedTokens and StakingCollection

### PR DESCRIPTION
**Pull request by danuio**
Resolves https://github.com/dapperlabs/flow-go/issues/5682.

- [x] Deploy `StakingProxy` contract 
- [x] Deploy `LockedTokens` contract
- [x] Deploy `StakingCollection` contract

### CODE REVIEW

File: fvm/bootstrap.go [hunk 1]
```diff
@@ -533,6 +542,61 @@ func (b *BootstrapProcedure) registerNodes(service, fungibleToken, flowToken flo
...
+		b.programs,
+	)
+	panicOnMetaInvokeErrf("failed to deploy StakingProxy contract: %s", txError, err)
+}
+
+func (b *BootstrapProcedure) deployLockedTokensContract(service flow.Address, fungibleTokenAddress,
+	flowTokenAddress flow.Address) {
+
+	publicKeys := make([]cadence.Value, 1)
+	encodedPublicKey, _ := flow.EncodeRuntimeAccountPublicKey(b.serviceAccountPublicKey)
```
Comment (jordanschalm) [inline fvm/bootstrap.go:560]: can we handle this error? a panic is fine

**Review by kc1116 — APPROVED** [review]

File: fvm/bootstrap.go [hunk 2]
```diff
@@ -637,6 +701,32 @@ transaction(amount: UFix64, recipient: Address) {
...
 
+const deployLockedTokensTemplate = `
+transaction(publicKeys: [[UInt8]]) {
+    
+    prepare(admin: AuthAccount) {
+        admin.contracts.add(name: "LockedTokens", code: "%s".decodeHex(), admin)
+
+        for key in publicKeys {
+            admin.addPublicKey(key)
+        }
```
Comment (jordanschalm) [inline fvm/bootstrap.go:712]: Now that this is deployed to the service account, we're just re-adding the key that's [already there](https://github.com/onflow/flow-go/pull/997/files#diff-f414ad868524efa32a2d6a821877f9d01b7948e26bece32df25664a8eb4709d8R560). I think we can remove this loop.
```suggestion
```

**Review by jordanschalm — APPROVED** [review]
Nice - I think we can remove the keys on the locked token account, otherwise LGTM 🎸

**Review by kc1116 — APPROVED** [review]

File: fvm/accounts_test.go [hunk 3]
```diff
@@ -1274,7 +1274,7 @@ func TestAccountBalanceFields(t *testing.T) {
 				assert.NoError(t, err)
 				assert.NoError(t, script.Err)
 
-				assert.Equal(t, cadence.UFix64(9999_5030), script.Value)
+				assert.Equal(t, cadence.UFix64(9999_5020), script.Value)
```
Comment (jordanschalm) [inline fvm/accounts_test.go:1277]: Hey @ramtinms this value changed after we deployed some extra contracts during execution state bootstrapping in this PR - is that expected and is it OK to update this expected value?

Comment (ramtinms) [reply fvm/accounts_test.go:1277]: yeah should be fine, it makes sense if we have deployed more contracts

**Pull request merged**